samples/javascript_nodejs/04.simple-prompt/bot.js

```diff
@@ -90,11 +90,11 @@ class MainDialog {
 
             // Show menu if no response sent
```

Comment:
Unrelated to this PR, can you require { ActivityTypes } from 'botbuilder' and use ActivityTypes.Message in line 72?

Reply:
done